packages/botonic-react/src/components/message.jsx

```diff
@@ -28,13 +29,13 @@ export const Message = props => {
   const textChildren = React.Children.toArray(children).filter(
```

Comment:
not sure if it would make sense to move this method within decomposeComponent

Reply:
This method it's used in the message component in order to display only the component without buttons and/or replies, it doesn't make any modification in the Component or the childrens. So I think it's okai to leave it here so we are using it only in the render of the Message component.

Reply:
ok